File: cours-filters-java/src/test/java/fr/isima/controllers/HelloControllerTest.java
package fr.isima.controllers;

import org.junit.Before;
import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class HelloControllerTest {

    private HelloController helloController;

    @Before
    public void setUp() throws Exception {
        this.helloController = new HelloController();
    }

    @Test
    public void hello_returnHello() throws Exception {
        assertThat(this.helloController.hello()).isEqualTo("Hello !");
    }
}

File: cours-filters-kotlin/src/test/kotlin/controllers/HelloControllerTest.kt
package controllers

import fr.isima.controllers.HelloController
import org.junit.Before
import org.junit.Test

import org.assertj.core.api.Assertions.assertThat

class HelloControllerTest {

    private val helloController: HelloController = HelloController()


    @Test
    fun hello_returnHello() {
        assertThat(this.helloController.hello()).isEqualTo("Hello !")
    }
}